dask/dataframe/core.py

```diff
@@ -33,11 +33,14 @@
 from .hashing import hash_pandas_object
 from .utils import (meta_nonempty, make_meta, insert_meta_param_description,
                     raise_on_meta_error, clear_known_categories,
-                    is_categorical_dtype, has_known_categories)
+                    is_categorical_dtype, has_known_categories, PANDAS_VERSION)
 
 no_default = '__no_default__'
 
-pd.computation.expressions.set_use_numexpr(False)
+if PANDAS_VERSION >= '0.20.0':
+    pd.core.computation.expressions.set_use_numexpr(False)
+else:
+    pd.computation.expressions.set_use_numexpr(False)
```

Comment:
Just a note to mention that after raising some concerns about backward compatibility pandas has added some more deprecation warnings and have made some more functions publicly available - e.g.
```python
pd.set_option('compute.use_bottleneck', False)
pd.set_option('compute.use_numexpr', False) 
```
...since anything under `core` is supposed to be private.